Update documentation.rst

rx9933 opened the pull request:
editing "check_hemisphere" function to return string (in typehints).

rx9933 committed:
Update documentation.rst

editing "check_hemisphere" function to return string (in typehints).

wjallen commented:
Hey Arushi - thanks for this pull request! If you change "string" to "str", I will accept this

rx9933 committed:
Update documentation.rst

changing function check_hemisphere function to return str instead of unknown type "string".

rx9933 commented:
I'm so sorry! I got MATLAB and Python syntax mixed up.

wjallen commented:
Thanks for the pull request!
